Automattic/vip-go-mu-plugins
# Use Jetpack Connection Pilot on VIP Migrations

## Description

One of the cleanup tasks that happens after a data migration is a reconnection to Jetpack, Akismet and VaultPress. There were some functions that implemented it and did very similar things to what Connection Pilot does. For that reason, we are getting rid of those additional functions and using Connection Pilot to perform the reconnection.

With this changes, we reduce the number of places and code paths that can happen to connect those components.

## Changelog Description

### Use Jetpack Connection Pilot on VIP Migrations

Jetpack, Akismet and VaultPress reconnection after a data migration is now powered by Connection Pilot.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Perform a data migration on a sandbox.
3. Check that Jetpack, Akismet and VaultPress are connected correctly.
